### PR TITLE
🌈 Add multi-asset progress bars

### PR DESCRIPTION
**Pull request by czunker**
Signed-off-by: Christian Zunker <christian@mondoo.com>

### CODE REVIEW

**Comment by czunker** [comment]
This requires an updated cnquery after https://github.com/mondoohq/cnquery/pull/771 is merged.

**Comment by czunker** [comment]
Some examples how the output looks like:

Screenshot from 2023-01-12 08-49-33
Screenshot from 2023-01-12 08-49-14
Screenshot from 2023-01-12 08-48-00
Screenshot from 2023-01-12 08-47-22

**Comment by czunker** [comment]
This currently only works in incognito mode, and needs a bugfix when working with upstream.

File: policy/scan/scan.proto [hunk 1]
```diff
@@ -47,6 +47,8 @@ message Job {
   // Vault configuration + credentials query
 	// Report type configuration
   ReportType report_type = 22;
+
+  int32 progress_num_assets = 23;
```
Comment (chris-rock) [inline policy/scan/scan.proto]: This should be removed here

Comment (czunker) [reply policy/scan/scan.proto]: Good catch. Thanks.

I removed it.

**Review by chris-rock — APPROVED** [review]
Thank you @czunker

**Pull request merged**